content/wasm-languages/zig.md

```diff
@@ -27,29 +27,103 @@ For the most part, write your Zig code as usual. When compiling, use the target
 
 Things we like:
 
-- The Zig compiler is so nice, we now use it to compile our C as well as our Zig.
+- The Zig compiler is so nice that we now use it to compile our C and Zig.
 
 Things we're not big fans of:
 
-- When we get Wasm-related errors, they can be really terse.
+- When we get Wasm-related errors, they can be really terse. However, since the original publishing of this page (around Zig 0.4.0), updates have been made to the error section of the official Zig documentation e.g. explaining how [Zig's error handling](https://ziglang.org/documentation/0.11.0/#toc-Common-errdefer-Slip-Ups) includes `defer` statements and `errdefer`, which triggers on block-exit errors. For example:
```

Comment:
I wouldn't bother discussing the publishing history - talk about the current situation.